style/modules/header.css

```diff
@@ -0,0 +1,47 @@
+.header {
+    min-width: 500px;
+    display: block;
+    display: flex;
+    align-items: center;
+}
+
+.header > *:not(.header--linkLogo) {
+    padding: 10px;
+    flex-grow: 1;
+    text-align: center;
+}
+
+.header--logo {
+    width: 100px;
+}
+
+.header--linkLogo {
+    flex-grow: 2;
+    margin-right: auto;
+}
+
+.header--link {
+    text-decoration: none;
+    font-size: 16px;
+    color: #7E7E7E;
+}
+
+.header--link:hover {
+    color: #113255;
+}
+
+.header--hamburger {
+    display: none;
+}
+
+@media only screen and (max-width: 1000px) {
+    .header--link,
+    .header .btnLink {
```

Comment:
if you make the changes I mentioned above, you can remove `.header .btnLink` from the targeting because they'll all be `header--link`s. Doing so should also remove some stray padding to the right of the hamburger menu icon (I didn't test this theory though.)